Makefile

```diff
@@ -87,17 +81,8 @@ endif
 	@echo "    test"
 	@echo "        This command will run the tests for the repository"
 	@echo "    "
-	@echo "    get-heroku-api-key"
-	@echo "        This command will run the heroku cli to get API KEY"
-	@echo "    "
-	@echo "    download-remote-db-heroku"
-	@echo "        This command will download the heroku remote Database"  sync-db-with-heroku
-	@echo "    "
-	@echo "    import-dump-to-db"
-	@echo "        This command will restore the dump to the local db"
-	@echo "    "
-	@echo "    sync-db-with-heroku"
-	@echo "        This command will synchronize the local db with the remote db on heroku. Careful! Some of your local data could be lost."
+	@echo "    update migrate=true data=remote"
```

Comment:
As funções get-heroku-api-key, download-remote-db-heroku, import-dump-to-db e sync-db-with-heroku foram removidas durante o merge. Elas ainda deveriam estar presentes no makefile